earth_enterprise/rpms/build.gradle

```diff
@@ -443,7 +448,11 @@ built from raster, vector, and location properties data.
     autoFindProvides = true
     autoFindRequires = true
 
+    requires('proj-devel') /* This dependency is not being picked up automatically by GeeRpm task */
+    requires('opengee-postgis', '1.5.8', GREATER | EQUAL)
```

Comment:
I don't think Fusion needs PostGIS. Why is it a dependency?

Reply:
It was a transient dependency before these changes (by way of opengee-common).  I did not want to remove this without a complete regression test - but perhaps we can in a follow-on issue.